src/main/java/org/sagebionetworks/web/client/widget/FullWidthAlert.java

```diff
@@ -156,6 +156,11 @@ public void setAlertType(AlertType type) {
     rerender();
   }
 
+  public void setOnClose(FullWidthAlertProps.Callback onClose) {
+    this.onClose = onClose;
+    rerender();
```

Comment:
provide ability to set a custom onClose callback, which will result in setting a local storage key for this case